Navaids/FIXES_Old.txt

```diff
@@ -28,3 +28,6 @@ PHILI N049.28.31.000 W007.01.17.000
 PIKOD N049.24.38.000 W005.15.59.000
 SUPAP N049.26.02.000 W005.49.22.000
 XAVAP N049.14.02.000 W006.08.35.000
+
+;withdrawn 12/2022
+BAKER N051.29.42.000 E000.17.53.190
```

Comment:
```suggestion
```

BAKER is not used anywhere (it was only on the N601) so it can be removed completely